locales/de/common.json

```diff
@@ -70,7 +73,7 @@
   "Skyrim Special Edition": "Skyrim Special Edition",
   "X-COM 2": "X-COM 2",
   "Associate with NXM URLs": "NXM URLs verwalten",
-  "Fix nxm link on Chrome (Requires Chrome to be closed)": "NXM-Links für Chrome reparieren (Chrome darf nicht laufen)",
+  "Fix nexusmods-links in Chrome (Only required for Chrome. Requires Chrome to be closed)": "NXM-Links für Chrome reparieren (Nur für Chrome benötigt, Chrome währenddessen darf nicht laufen)",
```

Comment:
Hier verstehe ich nicht warum du das "Only required for Chrome" hinzugefügt hast. Jemand der kein Chrome benutzt wird auch nicht unbedingt das bedüftnis haben auf diesen Button zu klicken.

Reply:
Ich hab am englischen Text nichts geändert (kann ich auch in der deutschen common.json gar nicht), ich hab nur gesehen, dass in der englischen Version anscheinend in der Zwischenzeit der Text verändert wurde, weswegen die "englische Seite" in common.json nicht mehr gestimmt hat und daher nicht erkannt wurde. Ich hab einfach die englische Version abgeschrieben und übersetzt, so wie es dasteht.

Reply:
Leider gibt es doch User die dann fragen "wo bleibt das Feature für Firefox?" und dann muss ich erklären dass sie es nicht brauchen...